ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -753,6 +753,11 @@ do
   cp "${item}" "${dir}/tls/${manifest##tls_}"
 done <   <( find "${SHARED_DIR}" \( -name "tls_*.key" -o -name "tls_*.pub" \) -print0)
 
+if [[ -f "${SHARED_DIR}"/99-swap-memory-enabled-swapaccount.yaml ]]; then
+    echo "Enable swap memeroy on nodes during installing cluster"
+    cp "${SHARED_DIR}"/99-swap-memory-enabled* "${dir}/openshift/"
```

Comment:
If we name the manifest yaml to `manifest_99-swap-memory-enabled-swapaccount.yaml`, the file can be included by the line 741 to 745 ?

Reply:
ah, so the file has to be placed in openshift dir, if right, can we develop a similar loop to manifest logic ?

Reply:
So the file named something like "openshift_<number>_xxxxx" can be copied from SHARED_DIR to <install-dir>/openshift ? Then remove the keywords "openshift_" from the file name?

Reply:
exactly

Reply:
Done